lib/src/chart/base/axis_chart/axis_chart_data.dart

```diff
@@ -25,19 +25,19 @@ abstract class AxisChartData extends BaseChartData {
   /// A background color which is drawn behind th chart.
   Color backgroundColor;
 
-  AxisChartData({
-    this.gridData = const FlGridData(),
-    FlBorderData borderData,
-    FlTouchData touchData,
-    this.axisTitleData,
-    this.rangeAnnotations = const RangeAnnotations(),
-    this.minX,
-    this.maxX,
-    this.minY,
-    this.maxY,
-    this.clipToBorder = false,
-    this.backgroundColor,
-  }) : super(borderData: borderData, touchData: touchData);
+  AxisChartData(
```

Comment:
Hi, please revert all reformat changes.

Reply:
What should I do? I just pull request, If you don't merge that I changed, you can just close without merge

Reply:
There are lot's of changes due to code formatting, they revert them and make a new commit, then I will review it.
Thanks!